XCUITests/FxScreenGraph.swift

```diff
@@ -1020,6 +1029,33 @@ extension MMNavigator where T == FxUserState {
     }
 }
 
+let NATIVE_EXECUTION    = Int32(0)
+let EMULATED_EXECUTION   = Int32(1)
+
+func processIsTranslated() ->Int32 {
+    var ret = Int32(0)
+    var size = ret.bitWidth
+    let result = sysctlbyname("sysctl.proc_translated", &ret, &size, nil, 0)
+    if result == -1 {
+        if (errno == ENOENT){
+          return 0
+        }
+        return -1
+      }
+    return ret
+}
+
+func processIsTranslatedStr() -> String {
+    switch processIsTranslated() {
+    case NATIVE_EXECUTION:
+        return "native"
+    case EMULATED_EXECUTION:
+        return "rosetta"
+    default:
+        return "unkown"
+    }
+}
+
```

Comment:
I'm not sure what these functions are doing, are they copied from somewhere? They will probably need comments

Reply:
what they are doing is to check which is the OS used. And so, depending on that the tests will use one way or the other to load a website. All this is a workaround to the issue mentioned [here](https://github.com/mozilla-mobile/firefox-ios/issues/9910#issue-1120710818)

Reply:
Ok. Thanks for adding a comment r+